### ISSUE TITLE
"backport" parTac to Coq

### ISSUE THREAD

**Issue by gares** [coq-community/vscoq#212]
It's an option if at some point we want to ditch the STM (move it to CoqIDE) and use a linear compiler but keep par:

**Comment by rtetley** [comment]
@gares is this something relevant to VsCoq 2 ?

**Comment by gares** [comment]
Yes, when I'll find the time to move the code from here to Coq this issue can be closed.
